graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+from copy import deepcopy
+
+from graphql import build_ast_schema
+from graphql.language import ast as ast_types
+from graphql.language.printer import print_ast
+import six
+
+from .utils import (
+    SchemaNameConflictError, SchemaStructureError, check_ast_schema_is_valid,
+    check_type_name_is_valid, get_query_type_name
+)
+
+
+MergedSchemaDescriptor = namedtuple(
+    'MergedSchemaDescriptor', (
+        'schema_ast',  # Document, AST representing the merged schema
+        'name_id_map',  # Dict[str, str], type name to id of the schema the type is from
+    )
+)
+
+
+def basic_schema_ast(query_type):
```

Comment:
I think it's best if we remove this function and just put the code inside `merge_schemas`. I don't think any other function will use it and it requires a lot of specific documentation.

Reply:
What if I put an underscore in front? I'm not totally sure about the balance between having long functions, and having helpers that are for very specific situations and are used only once.